src/manifest.ts

```diff
@@ -749,6 +776,25 @@ export class Manifest {
         }
       );
       return updatedPullRequest;
+    } else if (snoozed) {
+      // If unchanged, no need to push updates
+      if (snoozed.body === pullRequest.body.toString()) {
+        logger.info(
```

Comment:
I also think it would be worth pulling this logic into a helper, as the enclosing function is getting a bit long.